app/scripts/metamask-controller.js

```diff
@@ -1884,12 +1884,9 @@ export default class MetamaskController extends EventEmitter {
    * or the network "gas" currency, for the current selected network.
    *
    * @param {string} address - The address to fund.
-   * @param {string} amount - The amount of ether desired, as a base 10 string.
+   * @param {string} [amount] - The amount of ether desired, as a base 10 string.
    */
-  buyEth (address, amount) {
-    if (!amount) {
```

Comment:
This is also a functional change. Though... this function appears to be entirely unused as well, so I guess it doesn't matter.

Reply:
Yeah I removed this in a different PR, I'll update

Reply:
I removed this in #9236